src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -262,14 +264,17 @@ def _resampling_request_sender(self) -> Sender[ComponentMetricRequest]:
                 config=self._resampler_config,
             )
             self._resampling_actor = _ActorInfo(actor, channel)
+            self._resampling_actor.actor.start()
         return self._resampling_actor.channel.new_sender()
 
     async def _start(self) -> None:
         """Start the data pipeline actors."""
         if self._data_sourcing_actor:
-            await self._data_sourcing_actor.actor.start()
+            await self._data_sourcing_actor.actor.ready()
         if self._resampling_actor:
-            await self._resampling_actor.actor.start()
+            await self._resampling_actor.actor.ready()
+        if self._power_distributing_actor:
+            await self._power_distributing_actor.ready()
```

Comment:
I still don't see why do we need to have the `_start()` if we are starting all actors elsewhere anyway. I think we either remove `_start()` or we leave the eager starting in `_start()` and make users of this class call `_start()` before using it.

As I said, I would make the date pipeline a background service. If we only want a *temporary* fix in this PR, then I would only start in `_start()`.

With the power distributing actor is different because it start lazily only in the battery pool is used.

Reply:
I'm fine with that too, so go ahead and make everything lazy

Reply:
lazy is my specialty